lib/saml_idp.rb

```diff
@@ -86,8 +86,8 @@ def signature_namespace
         Namespaces::SIGNATURE
       end
 
-      def gather_errors(fingerprint, options = {})
-        signed_document.validate(fingerprint, false, options)
+      def gather_errors(cert, options = {})
+        signed_document.validate(cert, options, soft: false)
       rescue SamlIdp::XMLSecurity::SignedDocument::ValidationError => e
         { cert: options[:cert].serial.to_s, error_code: e.error_code }
```

Comment:
if we remove passing in `options[:cert]` we can probably change this to be `cert` right?
```suggestion
        { cert: cert.serial.to_s, error_code: e.error_code }
```

Reply:
updated in https://github.com/18F/saml_idp/pull/130/commits/243163a76d36ecaa037f9138836059ffca412014